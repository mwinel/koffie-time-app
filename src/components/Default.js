import React from "react";
import logo from '../logo.png'

const Default = () => {
  return (
    <div className="container my-5">
      <h1>koffie time app</h1>
      <img src={logo} alt="logo" width={40}/>
    </div>
  );
};

export default Default;
