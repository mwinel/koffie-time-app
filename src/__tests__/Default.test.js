import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import pretty from "pretty";

import Default from "../components/Default";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it("renders default component", () => {
  act(() => {
    render(<Default />, container);
  });
  expect(container.textContent).toBe("koffie time app");
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
