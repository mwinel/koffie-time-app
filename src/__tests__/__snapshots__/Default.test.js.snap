// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default component 1`] = `
"<div class=\\"container my-5\\">
  <h1>koffie time app</h1><img src=\\"logo.png\\" alt=\\"logo\\" width=\\"40\\">
</div>"
`;
